### DISCUSSION TITLE
data.entity.country field is not valid

### DISCUSSION THREAD

**Discussion by baldioli** [fattureincloud/api#212] [Q&A] [answered]
Inserendo "Paesi Bassi" come valore nella entity del campo Country ottengo l'errore riportato in oggetto anche le la scritta "Paesi Bassi" l'ho presa direttamente dalla chianata **List Countries**

**Comment by emajo** [comment, marked as answer]
Ciao @baldioli, ho provato a mettere "Paesi Bassi" come country e ha funzionato correttamente, ti consiglio di scrivere al nostro supporto clienti per andare a vedere nello specifico i log delle tue chiamata e individuare la problematica.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @baldioli sei riuscito a risolvere tramite supporto clienti?

**Comment by baldioli** [comment]
No, non li ho contattati. 
Ho risolto il problema contingente scrivendo Italia e poi andando a correggere dalla interfaccia di FattureInCluod.
Se mi ricapita contatterò il supporto clienti.
Per ora grazie
